src/vogsphere.html

```diff
@@ -15,15 +15,33 @@ <h3>Basics</h3>
             <input id="vogsphere__character-name" name="characterName" type="text" required>
         </div>
 
-        <div class="__input-group">
-            <label for="vogsphere__member-group">Member Group</label>
-            <select id="vogsphere__member-group" name="memberGroup" required>
-                <option value="">--Please choose an option--</option>
-                <option value="crew">Crew</option>
-                <option value="scientist">Scientist</option>
-                <option value="passenger">Passenger</option>
-            </select>
-        </div>
+        <!--Added radio buttons for member-group selection-->
```

Comment:
Could you please remove this comment? It's not needed and potentially confusing for people using this tool.